xpublish/rest.py

```diff
@@ -111,7 +138,16 @@ def get_dataset_from_plugins(self, dataset_id: str):
         return self._datasets[dataset_id]
 
     def setup_plugins(self, plugins: Optional[Dict[str, Plugin]] = None):
-        """Initialize and load plugins from entry_points"""
+        """Initialize and load plugins from entry_points unless explicitly provided
+
+        Parameters:
```

Comment:
I've seen that currently there are 4 occurences of 

```
Parameters:
```

and 4 occurences of 

```rst
Parameters
----------
```
in the docs. I think this should be consolidated. As a napoleon user, I would expect 

```rst
Parameters
----------
```
as the style of the sections in the docstrings. The same is true for the other section types (Returns, Raises, ...)